graphics/opencv4/Portfile

```diff
@@ -39,9 +39,20 @@ checksums           ${distname}${extract.suffix} \
                     sha256  03d9f7d777d8a62a5481934dbba2fd692b6cb43f1690a8546e45f9bdb1ee0863 \
                     size    90119002
 
+#------------------------------------------------------------------------------
+# Subport name used for staging files, etc.
+#
+# This is needed to support Python subports, whose names have no relation to
+# the actual physical file layout.
+#------------------------------------------------------------------------------
+set physical_subport_name \
```

Comment:
Why not just use `${name}` everywhere? The subport names are `${subport}`, so this is extra variable name is unnecessary and not consistent with standard Portfile practice, to the best of my knowledge.